hashtable.csv

```diff
@@ -10,3 +10,14 @@ code, name
 0x19, shake-256
 0x40, blake2b
 0x41, blake2s
+0x55, md5
+0x60, xxHash32
+0x61, xxHash64
+0x62, Spooky32
+0x63, Spooky64
+0x64, Spooky128
+0x65, Murmur3A
+0x66, Murmur3C
+0x67, Murmur3F
+0x75, crc32
+0x76, crc32-C
```

Comment:
Why is there a need for non-cryptographic hashes? I don't really think they have their place in the mutihash spec. I like the description in RFC 6920:

> The various formats are designed to support, but not require, a
> strong link to the referenced object, such that the referenced object
> may be authenticated to the same degree as the reference to it.

Unless there is a use /not/ to reference object such that they can be authenticated by the hash, you could of course add MD5, CRC-32 and so on, but I would stay away from it.